backend/sql/initial_schema.sql

```diff
@@ -0,0 +1,100 @@
+create table project
+(
+    project_id  bigserial     not null primary key,
+    name        varchar(256)  not null unique,
+    description varchar(1000) null
+);
+
+create table test
+(
+    test_id bigserial    not null primary key,
+    name    varchar(256) not null unique
+);
+
+create table subtest
+(
+    subtest_id      bigserial      not null primary key,
+    test_id         bigint         not null,
+    name            varchar(256)   not null unique,
+    -- Method
+    meth            varchar(10000) null,
+    confluence_link varchar(1000)  null,
+    constraint belongs_to_test
+        foreign key (test_id) references test (test_id)
+            on update cascade
+            on delete cascade
+);
+
+-- Corresponds to an EXCEL file
+create table experiment
+(
+    experiment_id bigserial                not null primary key,
+    project_id    bigint                   not null,
```

Comment:
I don't think so, yes relation between plate and subproject is not clear to me as well but from what I heard on the meeting: there might be some quirks with plate correspondence to subfolder but general case is that experiment corresponds to subfolder. We can re-listen to a meeting recording which @gromakovsky made.

>So project_id is not a property of experiment, but rather an analysis!
what? it sounds completely untruth to me. I mean it could be true that in one project used only specific analysis but there is no any explicit relation between them. 
I bet we can listen meeting recording/ask @gromakovsky understanding and we won't find proofs for this statement.

Reply:
Analysis in first version is just IC50, so how it's related to project at all? it's just a default thing which is used for any compound in any project as I understand.

Reply:
> We can re-listen to a meeting recording 

Please do, because the person who knows more said "compound ends up in a subproject".

> so how it's related to project at all? it's just a default thing which is used for any compound in any project as I understand.

 1. An instance of analysis is done over one compound and one compound only. 
 2. The same compound can be in different (sub-)projects.
 3. From (1) and (2) follows, that (sub-)projects do not track `experiment` *unless* you consider the same plate readout to *always* have compounds related to a single (sub-)project. This is where we have difference of opinions, but I'm certain that I'm right.

Reply:
I didn't understand whether an experiment (i. e. one `.xlsx` file) is always limited to one `project` or not.

Reply:
It's not, but we decided that it is.

Reply:
(For the purposes of MVP)